pkg/plugins/plugins.go

```diff
@@ -119,6 +119,9 @@ type JSONData struct {
 
 	// App Service Auth Registration
 	IAM *pfs.IAM `json:"iam,omitempty"`
+
+	// API Version: Temporary field while plugins don't expose a OpenAPI schema
+	APIVersion string `json:"apiVersion,omitempty"`
```

Comment:
we should maybe enforce:
https://github.com/kubernetes/apimachinery/blob/master/pkg/version/helpers.go#L34
```
"^v([\\d]+)(?:(alpha|beta)([\\d]+))?$"
```

Reply:
We could add this check in the plugin schema, as [we do for the plugin ID](https://github.com/grafana/grafana/blob/pluginApiVersion/docs/sources/developers/plugins/plugin.schema.json#L13). I consciously avoided adding it because it's something temporary / experimental but we could add it (also for external plugins to be able to define it). Thoughts?